README.md

```diff
@@ -172,6 +176,11 @@ All the Alpha2, Alpha3, Numeric and Name values have a corresponding enum in the
 Country group membership can be checked by calling the `isMemberOf` method, supplying the FQN of a class that implements the `GroupInterface`. Several country groups are available: BRICS, EEA, EFTA etc. Countries can also have subdivisions, which can be of several types: countries, provinces, etc.
 
 ```php
+use PrinsFrank\Standards\Country\CountryAlpha2;
+use PrinsFrank\Standards\Country\CountryAlpha3;
+use PrinsFrank\Standards\Country\CountryName;
```

Comment:
This namespace is not used in this snippet
```suggestion
```

Reply:
++